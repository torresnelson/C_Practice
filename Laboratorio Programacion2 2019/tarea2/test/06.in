# prueba de esta_ordenada, mezcla, ordenar
esta_ordenada
mezcla .
ordenar
imprimir_cadena
mezcla (1,a) (2,b) (3,c) .
insertar_al_final (10,j)
esta_ordenada
mezcla .
ordenar
imprimir_cadena
mezcla (1,a) (2,b) (3,c) .
final_cadena
insertar_antes (1,z)
esta_ordenada
ordenar
insertar_al_final (9,i)
esta_ordenada
ordenar
esta_ordenada
imprimir_cadena
inicio_cadena
insertar_segmento_despues (2,y) (5,x) (7,w) (8,v) .
esta_ordenada
imprimir_cadena
mezcla .
mezcla (6,f) (12,bb) (13,cc) .
mezcla (1,a) (2,b) (3,c) .
intercambiar 1 4
intercambiar 3 7
intercambiar 2 5
imprimir_cadena
esta_ordenada
ordenar
esta_ordenada
imprimir_cadena
Fin