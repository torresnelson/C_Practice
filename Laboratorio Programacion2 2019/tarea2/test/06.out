1>#  prueba de esta_ordenada, mezcla, ordenar.
2>cad ordenada.
3>
4>Ordenada.
5>
6>(1,a)(2,b)(3,c)
7>Insertado al final.
8>cad ordenada.
9>(10,j)
10>Ordenada.
11>(10,j)
12>(1,a)(2,b)(3,c)(10,j)
13>loc al final.
14>Insertado antes de loc.
15>cad ordenada.
16>Ordenada.
17>Insertado al final.
18>cad no ordenada.
19>Ordenada.
20>cad ordenada.
21>(1,z)(9,i)(10,j)
22>loc al inicio.
23>Segmento insertado despuÃ©s de loc.
24>cad ordenada.
25>(1,z)(2,y)(5,x)(7,w)(8,v)(9,i)(10,j)
26>(1,z)(2,y)(5,x)(7,w)(8,v)(9,i)(10,j)
27>(1,z)(2,y)(5,x)(6,f)(7,w)(8,v)(9,i)(10,j)(12,bb)(13,cc)
28>(1,z)(1,a)(2,y)(2,b)(3,c)(5,x)(7,w)(8,v)(9,i)(10,j)
29>Intercambio.
30>Intercambio.
31>Intercambio.
32>(7,w)(8,v)(10,j)(1,z)(2,y)(9,i)(5,x)
33>cad no ordenada.
34>Ordenada.
35>cad ordenada.
36>(1,z)(2,y)(5,x)(7,w)(8,v)(9,i)(10,j)
37>Fin.
